spec/factories/generic_work_change_sets.rb

```diff
@@ -0,0 +1,4 @@
+FactoryGirl.define do
+  factory :change_set, class: Hyrax::GenericWorkChangeSet do
```

Comment:
Do we need this?

Reply:
It's not something I see in the Valkyrie or Figgy apps.

Reply:
Not yet, I think we will need one based on where my current work is taking me but I can back it out of this PR if desired